.circleci/docker/requirements-ci.txt

```diff
@@ -86,11 +86,11 @@ mypy==0.960
 #Pinned versions: 0.960
 #test that import: test_typing.py, test_type_hints.py
 
-#networkx
+networkx==2.6.3
 #Description: creation, manipulation, and study of
 #the structure, dynamics, and functions of complex networks
-#Pinned versions: 2.0
-#test that import:
+#Pinned versions: 2.6.3 (latest version that works with Python 3.7+)
```

Comment:
requirements allow you to specify multiple version depending on Python version, for example see 
https://github.com/pytorch/pytorch/blob/2f0172bfc4c151cbc08d186b67f0969d7efdf953/.circleci/docker/requirements-ci.txt#L101-L102

Reply:
I'd prefer to keep it at 2.6.3 just in case there are important differences between the networkx versions